resources/definitions/os_discovery/netgear.yaml

```diff
@@ -27,6 +28,92 @@ modules:
                     index: 'boxServicesTempSensorTemperature.{{ $index }}'
                     low_limit: boxServicesNormalTempRangeMin
                     high_limit: boxServicesNormalTempRangeMax
+
+                # SFP-/Port-Temperature
+                -
+                    oid: boxServicesFiberPortsOpticsTable
```

Comment:
Please use the format MIB-NAME::OID rather than just OID. Looks `xos.yaml` for an example